web/app/mu-plugins/mitlib-secrets-widget/mitlib-secrets-widget.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Plugin Name: MITlib Secrets Widget
+ * Plugin URI: https://github.com/mitlibraries/mitlib-secrets-widget
+ * Description: A plugin to list the defined secrets (keys, not their values) coming from Pantheon's Terminus Secrets plugin
+ * Version: 0.1.2
+ * Author: Matt Bernhardt
+ * Author URI: https://github.com/matt-bernhardt
+ * License: GPL2
+ *
+ * @package MITlib Secrets Widget
+ * @author Matt Bernhardt
+ * @link https://github.com/mitlibraries/mitlib-secrets-widget
```

Comment:
We've not followed a consistent method for attributing authorship to local plugins, IIRC. I think Hattie and Philip both have their name on plugins they wrote, while I've tried to use just "MIT Libraries" - or list both the org and person.

Whatever we want to do, I think we should do it consistently, but this has been pretty far down the priority list of things to address thus far.

Reply:
Yeah. I agree with the inconsistent attribution as a "we can talk about it later" thing, but in this case I was referring to the `@link` which seems a bit wrong unless we intend to keep the old repo around longterm. This can _also_ be a later thing.

Reply:
Oh, yeah. Hmm. I've made https://mitlibraries.atlassian.net/browse/ENGX-212 to figure out an approach for this sort of metadata within a monorepo. I suspect that `@link` may not be a required field, so it could be better to just not use that field here.